lucille-core/src/main/java/com/kmwllc/lucille/core/Condition.java

```diff
@@ -11,15 +11,27 @@ public class Condition implements Predicate<Document> {
 
   private final List<String> fields;
   private final Set<String> values;
-  private final String operator;
+  private final Operator operator;
+
+  private enum Operator {
+    MUST, MUST_NOT;
+
+    public static Operator getOperator(String op) {
```

Comment:
Suggest renaming this to get()

Also, see this pattern where each enum value stores its string name: https://howtodoinjava.com/java/enum/java-enum-string-example/#5-reverse-lookup-get-enum-name-from-value

Reply:
did that